packages/devextreme/js/core/errors.js

```diff
@@ -253,29 +253,28 @@ export default errorUtils({
     /**
     * @name ErrorsCore.W0019
     */
-    W0019: 'DevExtreme: Valid license key not found.\n\n' +
-        'If you are using a trial version, you must uninstall all copies of DevExtreme once your 30 days trial period expires. For licensing-related information, please refer to the DevExtreme End User License Agreement: https://js.devexpress.com/EULAs/DevExtremeComplete/.\n\n' +
-        'To continue using DevExtreme in a project, you must purchase a license. For pricing/licensing options, please visit: https://js.devexpress.com/Buy/.\n\n' +
-        'If you have licensing-related questions or need help with a purchase, please email clientservices@devexpress.com. We will be happy to follow-up.\n\n',
+    W0019: 'DevExtreme: Unable to Locate a Valid License Key\n\n' +
+        'If you are using a 30-day trial version of DevExtreme, you must uninstall all copies of DevExtreme once your 30-day trial period expires. For terms and conditions that govern use of DevExtreme UI components/libraries, please refer to the DevExtreme End User License Agreement: https://js.devexpress.com/EULAs/DevExtremeComplete.\n\n' +
+        'To use DevExtreme in a commercial project, you must purchase a license. For pricing/licensing options, please visit: http://js.devexpres.com/Buy.\n\n' +
```

Comment:
> http://js.devexpres.com/Buy

Should it be https? (c) spotted by Vladimir Abadzhev

Reply:
Fixed